index.js

```diff
@@ -80,25 +80,25 @@ module.exports = {
 
     // Let's try to match https based repo url
     let match = url.match(
-      /https?:\/\/(www\.)?(github|gitlab).com\/(?<owner>[\w.-]+)\/(?<name>[\w.-]+)\.git*/
+      /https?:\/\/(www\.)?(?<server>[\w.-]+).com\/(?<owner>[\w.-]+)\/(?<name>[\w.-]+)\.git*/
     );
     if (match && match.groups && match.groups.owner && match.groups.name) 
```

Comment:
Wouldnt check also for match.groups.name?

Reply:
You are right!